t3f/ops.py

```diff
@@ -1191,3 +1192,44 @@ def renormalize_tt_cores(tt, epsilon=1e-8):
 
       return TensorTrainBatch(new_cores)
 
+
+def ones_like(tt):
+  """Constructs t3f.ones with the shape of `tt`.
+
```

Comment:
Not that the TT-ranks of the result are always 1 regardless of the TT-ranks of the input object.